File: APLICACIONSPA/ClientApp/src/app/liquidacion/models/paciente.ts
export class Paciente {
    identificacion: string;
    valorServicio: number;
    salario: number;
    tarifa: number;
    coPago: number;
    CalcularCopago(): void {
if(this.salario>2500000){
    this.tarifa= 0.20;
    this.coPago = this.tarifa*this.valorServicio;

}else{
    this.tarifa= 0.10;
    this.coPago= this.tarifa*this.valorServicio;
}
   }
}

File: Entity/Paciente.cs
using System;

namespace Entity
{
 public class Paciente
 {
 public string Identificacion { get; set; }
 public double valorServicio { get; set; }
 public double salario { get; set; }
 public double tarifa { get; set; }
 public double coPago { get; set; }
public void CalcularCopago(){
            if(salario>2500000){
                tarifa= 0.20;
                coPago=tarifa*valorServicio;
            }else{
                if(salario<=2500000){
                    tarifa=0.10;
                    coPago=tarifa*valorServicio;
                }
            }
        }
 }
}
